network/wsNetwork.go

```diff
@@ -1821,6 +1830,18 @@ func (wn *WebsocketNetwork) sendPeerConnectionsTelemetryStatus() {
 			InstanceName:         peer.InstanceName,
 			DuplicateFilterCount: peer.duplicateFilterCount,
 		}
+		// unwrap websocket.Conn, requestTrackedConnection, rejectingLimitListenerConn
+		var uconn net.Conn = peer.conn.UnderlyingConn()
+		for {
+			wconn, ok := uconn.(wrappedConn)
+			if !ok {
+				break
+			}
+			uconn = wconn.UnderlyingConn()
+		}
```

Comment:
This scares me, but I don't know what to prefer.  Imagine a low level net.Conn that implements UnderlyingConn() to return itself.  That seems like a moderately reasonable thing to do, but now you get an infinite loop.  Or perhaps returns `nil` if there's no underlyingConn.   You're kind of guessing how the lowest layer Conn will handle saying that there is no further underlyingConn.  I doubt this rule is enshrined in docs?